test/unit/tech/html5.test.js

```diff
@@ -178,3 +178,48 @@ test('native source handler canHandleSource', function(){
   // Reset test video canPlayType
   Html5.TEST_VID.canPlayType = origCPT;
 });
+
+if (Html5.supportsNativeTextTracks()) {
```

Comment:
these only run if we are using and support native text tracks, so, I'm only running the tests in those browsers as well because it'll fail otherwise.